coil-gif/README.md

```diff
@@ -10,23 +8,38 @@ To add GIF support, import the extension library:
 implementation("io.coil-kt.coil3:coil-gif:3.0.0-rc01")
 ```
 
-And that's it! The `ImageLoader` will automatically detect any GIFs using their file headers and decode them correctly.
+And that's it! The `ImageLoader` will automatically detect any supported files using their headers and decode them correctly.
 
 Optionally, you can manually add the decoder to your component registry when constructing your `ImageLoader`:
 
 ```kotlin
+// For Android
 val imageLoader = ImageLoader.Builder(context)
```

Comment:
We should introduce a common function like `AnimatedImageDecoderFactory` (not sure about the naming) that automatically returns the right decoder factory implementation for the platform/API level.